test_queue_job/tests/test_job.py

```diff
@@ -148,6 +148,20 @@ def test_worker_pid(self):
         job_a.set_pending()
         self.assertFalse(job_a.worker_pid)
 
+    def test_backend_pid(self):
+        """When a job is started, it gets the backend PID of the process."""
+        method = self.env["res.users"].mapped
+        job_a = Job(method)
+        self.assertFalse(job_a.db_backend_pid)
```

Comment:
```
  File "/home/bruno/buildout/14.0/parts/queue/test_queue_job/tests/test_job.py", line 155, in test_backend_pid
    self.assertFalse(job_a.db_backend_pid)
AttributeError: 'Job' object has no attribute 'db_backend_pid'
```

Reply:
The REAL problem was that I just did not update the tests to align with the changes from yesterday.
But they are fixed now